notebooks.go

```diff
@@ -471,6 +475,19 @@ func (s *server) NewNotebook(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	//Add Language
+	//Validate that the language format is valid (language[_territory])
+	match, err := regexp.MatchString("[[:alpha:]]{2}(_[[:alpha:]]{2})?$", req.Language)
+		if (err != nil || !match) {
+			var errLanguageFormat = errors.New("Error server language format")
```

Comment:
Make error message more clear, so someone reading logs in a console is likely to understand what occurred.